layers/+tools/pdf-tools/packages.el

```diff
@@ -0,0 +1,114 @@
+;;; packages.el --- pdf-tools Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: André Peric Tavares <andre.peric.tavares@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq pdf-tools-packages
+      '(
+        pdf-tools
+        ))
+
+(defun pdf-tools/init-pdf-tools ()
+  (use-package pdf-tools
+    :defer t
+    :mode (("\\.pdf$" . pdf-view-mode))
+    :config
+    (progn
+      (pdf-tools-install)
+      ;; TODO: Make `/', `?' and `n' work like in Evil
+      (evilified-state-evilify pdf-view-mode pdf-view-mode-map
+        ;; Navigation
+        "j"  'pdf-view-next-line-or-next-page
+        "k"  'pdf-view-previous-line-or-previous-page
+        "l"  'pdf-view-next-page-command
+        "h"  'pdf-view-previous-page-command
```

Comment:
I removed the `M-m` references.

@nixmaniack I don't use them too.  The thing is that ideally the user would never need to horizontally scroll, since adjusting the zoom or fitting the page width would make it unnecessary.

@syl20bnr What do you think? Should we mimic DocView's keybindings? Perhaps change both? I set more things to be accessible through the leader key, which is different from what DocView currently does.